spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConnectorUserAgentProvider.java

```diff
@@ -100,4 +101,13 @@ static Optional<String> getGcpRegion() {
   public String getUserAgent() {
     return USER_AGENT + " datasource/" + dataSourceVersion;
   }
+
+  @Override
+  public String getConnectorInfo() {
```

Comment:
why this is in JSON format? Can you please add some documentation for that? Alternatively, return a map (which is more generic) and let the client to convert it to JSON

Reply:
Added the comment for using JsonFormat.